packages/emf/backend/sirius-components-emf/src/main/resources/messages/sirius-web-emf.properties

```diff
@@ -9,7 +9,8 @@
 #
 # Contributors:
 #     Obeo - initial API and implementation
-################################################################################################
-INVALID_INPUT=Invalid input type, "{0}" has been received while "{1}" was expected
-UNEXPECTED_ERROR=An unexpected error has occurred, please contact the server administrator
-INVALID_NUMBER=The new value "{0}" is not a valid number
+#################################################################################
+INVALID_INPUT = Invalid input type, "{0}" has been received while "{1}" was expected
+UNEXPECTED_ERROR = An unexpected error has occurred, please contact the server administrator
+INVALID_NUMBER = The new value "{0}" is not a valid number
+UPPER_BOUNDARY_REACHED = Unable to create a new instance of "{0}" in feature "{1}" because it has reach its upper-bound cardinality."
```

Comment:
The final `"` is not neeed; it shows as part of the message.

Reply:
Done